desktop/cmp/dash/canvas/DashCanvasModel.js

```diff
@@ -86,6 +91,8 @@ export class DashCanvasModel extends HoistModel {
      * @param {number[]} c.margin - Between items [x,y] in pixels
      * @param {number} c.maxRows - Maximum number of rows permitted for this container
      * @param {number[]} c.containerPadding - Padding inside the container [x, y] in pixels
+     * @param {string} [c.emptyText] - text to display when the container is empty
+     * @param {string} [c.addViewButtonText] - text to display on the add view button
```

Comment:
This seems rather specific, in any case, might call it `addViewText`.  Might it be used elsewhere? Do we need it?

Reply:
I agree.  Only question is how important is it to be consistent with the old DashContainer here?  (This is the member name and doc used in the old DashContainerModel)

Reply:
👍🏻  -- Got it